src/browser/synthesis-ui.js

```diff
@@ -121,12 +135,20 @@ define((require, exports, module) => {
 
 
   const updateByWebViewAction = (state, id, action) =>
-    action instanceof Focusable.Focus ? showWebViewByID(state, id) :
-    action instanceof Focusable.Focused ? showWebViewByID(state, id) :
-    action instanceof WebView.Close ? closeWebViewByID(state, id) :
-    (action instanceof WebView.Open && !action.uri) ? createWebView(state) :
-    action instanceof WebView.SelectNext ? selectNext(state) :
-    action instanceof WebView.SelectPrevious ? selectPrevious(state) :
+    action instanceof Focusable.Focus ?
+      showWebViewByID(state, id) :
+    action instanceof Focusable.Focused ?
+      showWebViewByID(state, id) :
+    action instanceof WebView.Close ?
+      closeWebViewByID(state, id) :
+    (action instanceof WebView.Open && !action.uri) ?
```

Comment:
P.S.: I'm not entirely sure that `Open({uri: null})` even happens now that `TransitionToOpenWithFade` is used. If it is not maybe it makes sense to make `uri` mandatory on `Open`.